kits/youtube-creator-copilot/prompts/agentic-youtube-creator_llm-node_system.md

```
@@ -0,0 +1,7 @@
+You are an expert YouTube Strategist. Given a niche and a target audience, you must provide:
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)